.rubocop.yml

```diff
@@ -149,7 +149,7 @@ Style/IfUnlessModifier:
   Description: Favor modifier if/unless usage when you have a single-line body.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier
   Enabled: true
-  MaxLineLength: 100
+  MaxLineLength: 80
```

Comment:
Note that this won't affect any existing one-liners that are greater than 80 characters. Those are still valid. This will only prevent Rubocop from forcing you to use a one-liner unless it can fit in 80 characters. I think 80 is good for readability of one-liner conditionals.